contracts/campaignAirdrops/shared/CampaignAirdropsShared.sol

```diff
@@ -52,16 +52,16 @@ abstract contract CampaignAidropsShared is AirdropMerkleProof {
     function claimHandler(bytes32[] calldata _merkleProof) internal {
         if (hasClaimed[msg.sender]) revert AlreadyRedeemed();
         if (checkProof(_merkleProof, merkleRoot) == false) revert NotEligible();
-        if (msg.value != airbroCampaignFactoryAddress.claimFee()) revert InvalidFeeAmount();
 
-        (bool success, ) = airbroCampaignFactoryAddress.treasury().call{ value: msg.value }("");
+        hasClaimed[msg.sender] = true;
 
-        if (success) {
-            hasClaimed[msg.sender] = true;
+        if (msg.value != airbroCampaignFactoryAddress.claimFee()) revert InvalidFeeAmount();
+        (bool success, ) = airbroCampaignFactoryAddress.treasury().call{ value: msg.value }("");
 
-            emit Claimed(msg.sender);
-        } else {
+        if (!success) {
             revert FeeNotSent();
+        } else {
```

Comment:
You don't need else here because if(!success) is returning at the end. Makes code a bit cleaner

Reply:
Good point, will fix this